galata/playwright.config.js

```diff
@@ -31,7 +28,12 @@ module.exports = {
     {
       name: 'jupyterlab',
       testMatch: 'test/jupyterlab/**',
-      testIgnore: '**/.ipynb_checkpoints/**'
+      testIgnore: '**/.ipynb_checkpoints/**',
+      use: {
+        contextOptions: {
+          permissions: ['clipboard-read', 'clipboard-write']
+        }
```

Comment:
The alternative - as used in https://github.com/jupyterlab/jupyterlab/pull/14345 - is to enable them on per-test basis:

```js
const browserContext = page.context();
await browserContext.grantPermissions(['clipboard-read']);
```

I guess either way is fine.